ccdproc/core.py

```diff
@@ -1740,6 +1741,94 @@ def _sigma_mask(baseline, one_sigma_value, lower_sigma, upper_sigma):
     return mask
 
 
+def bitmask2mask(bitmask, ignore_bits=0, flip_bits=None):
```

Comment:
I think `bitmask` (or `bitflags`) are more commonly used than `bitfield` so I think we should expose it under that name. however does anyone have a better idea for this function?